cfgov/paying_for_college/disclosures/scripts/nat_stats.py

```diff
@@ -18,7 +19,7 @@ def get_bls_stats():
     try:
         with open(BLS_FILE, 'r') as f:
             data = json.loads(f.read())
-    except FileNotFoundError:
+    except builtins.FileNotFoundError:
```

Comment:
Curious why this is needed. Isn't `FileNotFoundError` valid on its own? For example, [Django seems to use it that way frequently.](https://github.com/django/django/search?q=filenotfounderror&unscoped_q=filenotfounderror)

Reply:
This surprised me too. Faulty linter.
Fixed.